README.md

```diff
@@ -6,7 +6,13 @@ TODO write some useful stuff.
 
 ## Installation
 
-TODO write some useful stuff.
+1. Download `passafari` repo
+2. Sign up to be a Safari Dev
+2. Obtain a cert: https://developer.apple.com/membercenter/index.action
```

Comment:
Apple "simplified" their program and killed the free Safari Extension development. Now we have to pay $100/year for their Developer Program. RIP Apple. Read https://medium.com/@honestbleeps/what-apple-gives-you-for-100-as-a-safari-extension-developer-and-why-reddit-enhancement-suite-6e2d829c2e52 for details.